algorithms/sorting/insertionsort_1.java

```diff
@@ -0,0 +1,50 @@
+import java.io.*;
```

Comment:
Nice job!
Please, change the name to lowercase to keep the standard of other solutions.
A nice name would be `insertion_sort_part_1.java` to keep standard with the other language versions that already is in this repo.
Please, fix the indentation and remove whitelines too.